gcs/CHANGES.md

```diff
@@ -1,4 +1,8 @@
-### 2.2.6 - 2022-XX-XX
+### 2.2.6 - 2022-04-14
+
+1. Improve trace logging for gRPC channels.
+
+2. Bug fixes.
```

Comment:
We should mention bug fixes in detail as the write one was important.

@mprashanthsagar should be able to give summary for here.